resources/views/building/show.blade.php

```diff
@@ -0,0 +1,84 @@
+@extends('layouts.app')
+@section('title', $building->title)
+
+@section('content')
+
+@include('components.header')
+
+<div class="container-fluid py-3 px-3 bg-light border-bottom vh-38"> 
+    <div class="row">
+    	<div class="col-sm-4 order-sm-2 text-sm-right">
+        	<button class="btn btn-outline-dark btn-lg mb-2">MAPA</button>
+     	</div>
+        <div class="col-sm-8 order-sm-1">
+            @include('components.tags', ['tags' => $building->tags])
+        </div>
+	</div>
+</div>
+
+<div class="container-fluid py-5 px-3">	
+    <div class="row">
+
+        <div class="col-md-4 order-md-2">
+            <h2 class="mb-3 ls-2">
+                {{ $building->title }}
+            </h2>
+
+            <ul class="list-group list-group-flush border-top border-bottom mb-4">
+              <li class="list-group-item px-0 py-2">
+                  <b>{{ __('building.current_function') }}</b><br>
+                  {{ $building->current_function }}
+              </li>
+              <li class="list-group-item px-0 py-2">
+                  <b>{{ __('building.address') }}</b><br>
+                  {{ $building->location_street }}, {{ $building->location_city }}
```

Comment:
Just in case there's no `location_street`
```suggestion
                  {{ implode(', ', [$building->location_street, $building->location_city] }}
```